src/main/java/com/internet/mixshop/model/ShoppingCart.java

```diff
@@ -0,0 +1,34 @@
+package com.internet.mixshop.model;
+
+import java.util.List;
+
+public class ShoppingCart {
+    private long id;
```

Comment:
```suggestion
    private Long id;
```

add getter and fix setter for this field

Reply:
done